src/sentry/api/endpoints/project_ownership.py

```diff
@@ -28,17 +30,16 @@ def validate_raw(self, attrs, source):
 
         schema = dump_schema(rules)
 
+        owners = {o for rule in rules for o in rule.owners}
```

Comment:
I can never remember precedence rules here so thanks for clearly naming this, at least